databricks/sdk/_base_client.py

```diff
@@ -68,6 +69,7 @@ def __init__(self,
         :param extra_error_customizers:
         :param debug_headers: Whether to include debug headers in the request log.
         :param clock: Clock object to use for time-related operations.
+        :param streaming_buffer_size: The size of the buffer to use for streaming responses (default: 1MB).
```

Comment:
Nit: isn't the default value included in the docs already? Less risk for it to go out of sync.

Reply:
👍